plugins/baseball-scoreboard/manager.py

```
@@ -93,6 +93,16 @@ def __init__(
 
         self.logger = logger
 
+        # Resolve timezone: plugin config → global config → UTC.
+        # Inject into self.config so all sub-components (scroll display, game
+        # renderer, etc.) can read it via config.get('timezone').
+        if not self.config.get("timezone"):
+            try:
+                global_tz = cache_manager.config_manager.get_timezone()
+            except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)